plugin/color.py

```diff
@@ -45,11 +43,26 @@ def is_applicable(cls, _settings):
         disabled = 'colorProvider' in settings.disabled_capabilities
         return is_supported and not disabled
 
+    def initialize(self):
+        session = session_for_view(self.view)
+        if session is None:
+            # not yet initialized, try again
+            sublime.set_timeout_async(self.initialize, 400)
```

Comment:
Will this not create an endless loop if no session is ever created?
Hard to guess how long it takes for a session to start up, perhaps some servers take longer than 400ms.
Perhaps we can hook up to another view event instead of this set_timeout for re-trying initialisation?